setup/setup_env.py

```
@@ -0,0 +1,199 @@
+import time
+
+VENV_DIR = 'venv'
+LEGACY_VENV_DIR = 'gradio_venv_gpu'
+
+def run_command(cmd, description=""):
+    """Run a command and return success status"""
+    print(f"\n🔄 {description}")
+    print(f"Command: {cmd}")
+
+    try:
+        result = subprocess.run(cmd, shell=True, capture_output=True, text=True, timeout=300)
```

Ruff (0.13.1)

`subprocess` call with `shell=True` identified, security issue

(S602)